pkgs/native_assets_builder/test/data/cyclic_package_1/pubspec.yaml

```diff
@@ -16,4 +16,4 @@ dependencies:
   yaml_edit: ^2.1.0
 
 dev_dependencies:
-  lints: ^2.0.0
+  lints: ^3.0.0
```

Comment:
I was thinking that the tests could potentially also be `dart_flutter_team_lints`.

However, it's likely that these test projects will be used as "examples" by people at some point. And for examples we should probably stick with package:lints.

Reply:
Yeah I think for the tests, either stick to `lints` or remove the lints dependencies from them. Intentional examples should stick to `lints`.